vsphere/vm-http-server/main.tf

```
@@ -0,0 +1,35 @@
+provider "vsphere" {
```

Let's add the terraform version you want to use here, preferably 0.13.3:
 ```suggestion
terraform {
  required_version = "0.13.3"
}
 
provider "vsphere" {
```

---

Maybe you should pin `vsphere` provider version as well.
If you use tf13, the new format for `required_providers` is recommended, but not a must. See [here](https://www.terraform.io/docs/configuration/provider-requirements.html#requiring-providers)